README.md

```diff
@@ -31,6 +31,8 @@ Use the `access_token` from Chrome to do requests
   "picture_url": "<profile_picture_url_from_google>",
   "hours_start": <hour_to_begin_scheduling_tasks (initially null)>,
   "hours_end": <hour_to_end_scheduling_tasks (initially null)>,
+  "primary_calendar": "<primary_calendar_of_user>",
+  "relevant_calendars": "<relevant_calendars_of_user>",
```

Comment:
Can you change this to "relevant_calendars": "<comma_separated_relevant_calendars_of_user>"